src/pages/api/updateMarks.js

```diff
@@ -0,0 +1,111 @@
+const fs = require('fs').promises;
+const path = require('path');
+const process = require('process');
+const { authenticate } = require('@google-cloud/local-auth');
+const { google } = require('googleapis');
+import { PrismaClient } from '@prisma/client';
+
+const prisma = new PrismaClient();
+
+const SCOPES = ['https://www.googleapis.com/auth/spreadsheets.readonly'];
+
+const TOKEN_PATH = path.join(process.cwd(), 'token.json');
```

Comment:
just use env for that

Reply:
no, it must be in the code as google api docs said